.isort.cfg

```diff
@@ -2,3 +2,4 @@
 profile = black
 combine_as_imports = True
 line_length = 79
+src_paths = .
```

Comment:
this is better

Reply:
it's overrideable + it correctly determines the actual first-party package dirs using the same logic that python would